### ISSUE TITLE
03_linreg_starter will be overwrite loss

### ISSUE THREAD

**Issue by xiaosen-wang** [chiphuyen/tf-stanford-tutorials#138]
_, loss = ########## TO DO ############
            total_loss += loss
Here the loss variable name will overwrite the loss function so that in next epoch there will be a error "Can not convert a float32 into a Tensor or Operation."

**Comment by joseph-d-p** [comment]
@jiangchengyike  rename it to `loss_`
